Tests/CentralManagerUtilsTests.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import XCTest
+@testable import AsyncBluetooth
+
+class CentralManagerUtilsTests: XCTestCase {
+    func testIsBluetoothReady() throws {
+        XCTAssert(true)
```

Comment:
Let's remove this assert.

Reply:
DOH! Thanks for catching that. It is removed now